### PR TITLE
Feature/material ui update

### PR DESCRIPTION
**Pull request by seleregb**
#17 

1. Updated material-ui to latest and updated usage
2. Couldn't fully format the Learn Page

### CODE REVIEW

File: source/client/__tests__/notification/notification.reducer.spec.js [hunk 1]
```diff
@@ -1,39 +1,53 @@
-import { DISMISS_NOTIFICATION, SEND_NOTIFICATION } from "../../application/notification/notification.types"
-import { notificationReducer } from "../../application/notification/notification.reducer"
+'use strict';
```
Comment (devshawn) [inline source/client/__tests__/notification/notification.reducer.spec.js]: Hey -- great, great, great work on this! This is great experience on upgrading something with breaking changes. A lot of this looks good, but you still have to merge in the latest master. For example, this test in notification.reducer.spec.js, is being replaced with stuff that used to be in master but isn't anymore.

Comment (seleregb) [reply source/client/__tests__/notification/notification.reducer.spec.js]: Okay. I'll get on that :smile:

Comment (seleregb) [reply source/client/__tests__/notification/notification.reducer.spec.js]: I've done the merge

**Comment by clayreimann** [comment]
Is this ready? I'd like to make the site work better on mobile devices, but I don't want to go messing with the layout until it's been updated.

**Comment by seleregb** [comment]
> @devshawn Is this ready? I'd like to make the site work better on mobile devices, but I don't want to go messing with the layout until it's been updated.

Yes its ready

**Pull request closed without merging**